src/app/components/StartScreen/StartScreen.tsx

```diff
@@ -0,0 +1,80 @@
+import axios, { AxiosResponse } from 'axios';
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+
+interface IGame {
+    displayName: string;
+    id: string;
+}
+
+const StartScreen: React.FunctionComponent = () => {
+    const history = useHistory();
+    const [games, setGames] = React.useState<IGame[]>([]);
+    const [chosenIndex, setChosenIndex] = React.useState(0);
+
+    React.useEffect(() => {
+        axios
+            .get('/games')
+            .then((response: AxiosResponse<{ gamesConfig: IGame[] }>) => {
+                setGames(response.data.gamesConfig);
+            })
+            .catch((error: any) => {
+                alert(error);
+            });
+    }, []);
+
+    return (
+        <div className={'startscreen'}>
+            <h1 children={'Code in the Dark, by Bekk'} />
+
+            <div className={'startscreen__konfigurasjon'}>
+                <div className={'startscreen__konfigurasjon--valg'}>
+                    <h3>Velg spill og trykk START</h3>
```

Comment:
Her står det trykk START, mens teksten for å opprette et spill heter "Opprett spill"